public/content/documentation/web/component/link.md

```diff
@@ -6,75 +6,485 @@ How to test a link
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
+How to test a link
 
 1. Test keyboard only, then screen reader + keyboard actions
 
-   - Skip-links: Focus moves directly to the header or navigation
-
-   - Tab: Nothing, headings are not focusable unless they are actionable
-
-   - Arrow-keys: Headings are browsed
+   - Tab: Focus visibly moves to the link.
+   - Enter: Activates the link.
 
 2. Test mobile screenreader gestures
 
-   - Swipe: Focus moves directly to the header or navigation
-
-   - Doubletap: This typically activates most elements
+   - Swipe: Focus moves to the link
+   - Doubletap: Activates the link
 
 3. Listen to screenreader output on all devices
 
-   - It is discoverable with screenreader shortcuts as header/banner landmark
+   - Name: Its purpose is clear
+   - Role: It identifies itself as a link
 
-   - Group: It typically contains the name and primary navigation of the website
 
 Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/link](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/link)
 
 ## Gherkin
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
+How to test a link
 
-GIVEN THAT I am on a page with a header landmark
+GIVEN THAT I am on a page with a link
 
 1. Keyboard for mobile & desktop
 
-   - WHEN I use the tab key to enter the web browser window I SEE focus is strongly visually indicated on interactive components
+   - WHEN I use the tab key to move focus to a link I SEE focus is strongly visually indicated
+   - THEN when I use the enter key to activate the link I SEE my browser goes somewhere
 
 2. Desktop screenreader
 
-   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND 
+   - I use the tab key to move focus to a link
+      - I HEAR Its purpose is clear
```

Comment:
All "I HEAR" statements have capital "I"s after, in both step 2 and 3. Let's lowercase them for consistency/clarity.